docs/changelog.md

```diff
@@ -12,6 +12,9 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/android-components/blob/master/.config.yml)
 
+* **browser-engine-gecko**, **browser-engine-gecko-beta**, **browser-engine-gecko-nightly**
+  * 🚒 Bug fixed [issue #8431](https://github.com/mozilla-mobile/android-components/issues/8431) SpeculativeSessionObserver is swallowing events meant to be for EngineObserver.
```

Comment:
nit: This is describing the internal detail. Can we describe what it fixes from the point of the view of the consumer? :)

Reply:
Should you be 😴? 🤔

Reply:
Thanks, anyways :)